dirsrvtests/tests/suites/clu/schema_test.py

```diff
@@ -52,12 +52,11 @@ def test_origins_with_extra_parenthesis(topo):
     attr_result = schema.query_attributetype(ATTR_NAME, json=True)
 
     # Verify the x-origin value is correct
-    assert attr_result['at']['x_origin'][0] == X_ORG_VAL
+    assert attr_result['at']['x_origin'][0][0] == X_ORG_VAL
```

Comment:
This just seems wrong to me.  I recently added this test case to test the fix for the embedded parenthesis, and to have this double nested subscription does not look right.  What does the dict look like to require this?  I'm just wondering if this is something that needs to be fixed in lib389 instead (maybe a regression was introduced in lib389)?

Reply:
Right... I am not sure what made me do the fix like this. 
Please, check another fix :)